pkg/controller/networkpolicy/clustergroup.go

```diff
@@ -206,23 +206,27 @@ func (n *NetworkPolicyController) syncInternalGroup(key string) error {
 		return nil
 	}
 	grp := grpObj.(*antreatypes.Group)
+	if grp.SourceReference.Namespace != "" {
+		// Sync the Group as a Namespaced Group.
+		return n.syncNamespacedInternalGroup(grp)
```

Comment:
I feel a bit weird about this appearing in clustergroup.go. Maybe provide a higher level wrapper in crd_utils and rename this to `syncClusterInternalGroup`?

Reply:
done